spec/loadingANDcontactUSpages/LoadingANDcontactUS.txt

```diff
@@ -0,0 +1,13 @@
+Verify user is redirected to 'Contact us' page after clicking on 'Contact us' button on Landing page
+Verify that user can choose file in 'Attach File' field
+Verify that user can change 'Subject Heading' dropdown menu
+Verify that user can send a message with entered valid data
+Verify that user can't send a message without entered data
```

Comment:
what is valid and what is invalid data?
what happens on entering invalid data? errors shown?